DatadogSessionReplay/Sources/Writer/Models/SRDataModels+UIKit.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  File.swift
+//  
+//
+//  Created by Maciej Burda on 13/04/2023.
+//
+
+import UIKit
+
+extension SRTextPosition.Alignment {
+    /// Custom initializer that allows transforming UIKit's `NSTextAlignment` into `SRTextPosition.Alignment`.
+    /// Must be called on the main thread due to `UIApplication.shared.userInterfaceLayoutDirection`.
+    init(textAlignment: NSTextAlignment) {
+        self.vertical = .center
+        switch textAlignment {
+        case .left:
+            self.horizontal = .left
+        case .center:
+            self.horizontal = .center
+        case .right:
+            self.horizontal = .right
+        case .justified:
+            self.horizontal = .left
+        default:
+            if UIApplication.shared.userInterfaceLayoutDirection == .leftToRight {
+                self.horizontal = .left
+            } else {
+                self.horizontal = .right
+            }
+        }
```

Comment:
blocking/ We're not yet handling RTL languages and adding their support is far more than this `default` case (it will require more changes in the SDK, support to custom fonts and set of enhancements in the player). 

I don't see any value in "mimicking" RTL if it doesn't bring anything further (won't work well). In fact, we're making our life harder with this code as now it _**"Must be called on the main thread"**_. That threading constraint could be easily removed if we invest more effort: capture the layout direction on the main thread and propagate it along with the recording context down to wireframe builders (background thread). Do we want to pay this effort now given that we're unable to fully support RTL?

Reply:
Sounds good, I'll add a ticket 🙌